CMakePresets.json

```
@@ -45,8 +45,28 @@
       "cacheVariables": {
         "CMAKE_BUILD_TYPE": "RelWithDebInfo"
       }
-    }
-  ],
+    },
+    {
+      "name": "linux-default",
+      "displayName": "Linux Debug",
+      "description": "Sets Ninja generator, compilers, build and install directory, debug build type",
+      "generator": "Ninja",
+      "binaryDir": "${sourceDir}/out/build/${presetName}",
```

Please use the same structure as windows
```suggestion
      "binaryDir": "${sourceDir}/build/${presetName}",
```